docs/ga-requirements.md

```diff
@@ -0,0 +1,51 @@
+P1 (e.g. cannot GA without these):
+* End-to-end tests
+  * OTLP, Jaeger and Zipkin
+  * Spring Boot and Wildfly
+    * (Wildfly chosen due to common javaagent issues around jboss modules and jboss logging)
+  * Java 8, 11, and the latest Java version
+* Benchmarking
+  * Runtime overhead benchmark
+  * Startup overhead benchmark
+* Implement all required semantic attributes
+  * Non-required semantic attributes are nice to have, but not required for GA
+* Basic configuration points
+  * Add custom auto-instrumentation
+  * Ability to build "custom distro"
+* Documentation (tag "documentation" minus tag "contributor experience")
+  * All configuration options
+    * Standard OpenTelemetry SDK + Exporter configuration options
+    * Auto-instrumentation configuration options (e.g. disable/enable, peer.service mapping)
+  * For each instrumentation
+    * Document any instrumentation-specific configuration
+  * Document the basic configuration points (above)
+    * How to write your own auto-instrumentation
+      * (much of this can be shared with contributor documentation below)
+    * How to build your own "custom distro"
+  * How to troubleshoot (start documenting common issues somewhere)
+* Library (manual) instrumentations for libraries commonly used with Spring
```

Comment:
I am not sure this is hard requirement for GA

Reply:
I'd like to have some kind of hard requirement for manual instrumentation, in order to show our commitment to supporting both auto and manual instrumentation.

Reply:
Not sure if it matters much, but FWIW, the work for extracting manual instrumentation for okhttp and grpc seems very small compared to Spring which is a lot of new instrumentation, though @mabdinur is doing a great job tackling it :)